prac_05/state_names.py

```diff
@@ -1,21 +1,18 @@
 """
 CP1404/CP5632 Practical
 State names in a dictionary
-File needs reformatting
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
 STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
-# print(STATE_NAMES)
 
-state = input("Enter short state: ").upper()
-while state != "":
-    if state in STATE_NAMES:
-        print(state, "is", STATE_NAMES[state])
+full_state = input("Enter short state: ").upper()
+while full_state != "":
+    if full_state in STATE_NAMES:
+        print(full_state, "is", STATE_NAMES[full_state])
     else:
         print("Invalid short state")
-    state = input("Enter short state: ").upper()
+    full_state = input("Enter short state: ").upper()
```

Comment:
Think I'm partly to blame for this one saying 'state_long' might get confused for number type 'Long'.
Maybe use 'state_short_desc' and 'state_long desc'.
And yes as Lindsey said... this should be:
       `state_short_desc = input("Enter short state: ").upper() `